### PR TITLE
Welcome page

### LINKED ISSUE #1: Deployment Environment
#### As a developer I want a continuous deployment environment setup for my new Rails application.

Acceptance criteria:
- [x] The latest version of both Ruby and Rails are used both locally and on
  Heroku.
- [x] [GitHub](https://github.com/AppFolioOnboarding) is used for source
  control.
- [x] [CircleCI](https://circleci.com/) is used for Continuous Integration.
- [x] Heroku is used to deploy the code with puma as the application server.
- [x] The staging application automatically deploys whenever the master branch
  is updated.
- [x] The staging application automatically deploys only after all tests pass
  on CircleCI.
- [x] Pull requests are automatically deployed as review applications.
- [x] The staging application deployment can be promoted to the production
  application deployment.
- [x] Every available controller action has corresponding tests.
- [x] [Pronto](https://github.com/mmozuras/pronto) is used to help with code
  style consistency.

Discussion Topics:
- [x] Where can we discover how long it takes to return an HTTP response?
- [x] When should we commit our code?

References:
* https://rvm.io/rvm/install#try-out-your-new-rvm-installation
* http://guides.rubyonrails.org/getting_started.html
* http://guides.rubyonrails.org/testing.html#functional-tests-for-your-controllers
* https://devcenter.heroku.com/articles/getting-started-with-rails4
* https://devcenter.heroku.com/articles/pipelines
* https://github.com/bbatsov/ruby-style-guide

### PR DESCRIPTION
**Pull request by jeremyjpark**
Closes #1

### CODE REVIEW

**Comment by jeremyjpark** [comment]
Judging from what I heard earlier, I should also be testing to see if the content of the page is actually what I wanted it to be and not just a 200 http response.

File: app/helpers/welcome_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module WelcomeHelper
+end
```
Comment (scuendet) [inline app/helpers/welcome_helper.rb]: If you don't need it now, you can probably delete this file.

File: test/controllers/welcome_controller_test.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class WelcomeControllerTest < ActionController::TestCase
+  test "should get index" do
+    get :index
+    assert_response :success
```
Comment (scuendet) [inline test/controllers/welcome_controller_test.rb]: Additionally to that, you could use `assert_select` to test the content of your page.

Comment (jeremyjpark) [inline test/controllers/welcome_controller_test.rb]: I've added tests to check if the title is correct and if the header's content exists and is correct.

Other tests that might be relevant are tests to make sure that unwanted content isn't there, but that seems impractical?

Comment (scuendet) [inline test/controllers/welcome_controller_test.rb]: I think that's good for now, this page is pretty simple.

File: app/assets/stylesheets/welcome.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the welcome controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (scuendet) [inline app/assets/stylesheets/welcome.scss]: This is basically an empty file, so you don't need to include it at this time.

**Comment by scuendet** [comment]
Looks good.

**Comment by jeremyjpark** [comment]
I'll remove the .scss file and merge then!

**Pull request merged**